### PR TITLE
Add Panel 1.5 release blog post

### PR DESCRIPTION
**Pull request by philippjfr**
Adds blog post for Panel 1.5 release.

### CODE REVIEW

File: posts/panel_release_1.5/index.ipynb [hunk 1]
```diff
@@ -0,0 +1,2259 @@
...
+    "\n",
+    "### Introducing ESM Components\n",
+    "\n",
+    "The new component baseclasses [`JSComponent`](https://panel.holoviz.org/reference/custom_components/JSComponent.md), [`ReactComponent`](https://panel.holoviz.org/reference/custom_components/ReactComponent.md) and [`AnyWidgetComponent`](https://panel.holoviz.org/reference/custom_components/AnyWidgetComponent.md) build on a technology called ECMA script modules or ESM modules for short. These make it much simpler to build reusable modules that could easily import other libraries. Specifically they introduced import and export specifiers, which allow developers to import other libraries and export specific functions, objects and classes for the consumption of others.\n",
```
Comment (jbednar) [inline posts/panel_release_1.5/index.ipynb:1200]:
```suggestion
    "The new component baseclasses [`JSComponent`](https://panel.holoviz.org/reference/custom_components/JSComponent.md), [`ReactComponent`](https://panel.holoviz.org/reference/custom_components/ReactComponent.md) and [`AnyWidgetComponent`](https://panel.holoviz.org/reference/custom_components/AnyWidgetComponent.md) build on a technology called ECMAScript modules or ESM modules for short. These make it much simpler to build reusable modules that could easily import other libraries. Specifically they introduced import and export specifiers, which allow developers to import other libraries and export specific functions, objects and classes for the consumption of others.\n",
```

File: posts/panel_release_1.5/index.ipynb [hunk 2]
```diff
@@ -0,0 +1,2259 @@
...
+    "\n",
+    "<img src=\"./images/collage_1.5.png\" width=\"100%\"></img>\n",
+    "\n",
+    "## What is Panel?\n",
+    "\n",
+    "Panel is an open-source Python library that lets you **easily build powerful tools, dashboards and complex applications entirely in Python**. It has a batteries-included philosophy, putting the PyData ecosystem, powerful data tables and much more at your fingertips. High-level reactive APIs and lower-level callback based APIs ensure you can quickly build exploratory applications, but you aren’t limited if you build complex, multi-page apps with rich interactivity. Panel is a member of the [HoloViz](https://holoviz.org) ecosystem, your gateway into a connected ecosystem of data exploration tools.\n",
+    "\n",
+    "## New release!\n",
+    "\n",
+    "We are very pleased to announce the 1.5.0 release of Panel! While technically a minor release hugely expands the scope of what is possible in Panel. A high-level overview of the most important features:\n",
```
Comment (jbednar) [inline posts/panel_release_1.5/index.ipynb:1171]:
```suggestion
    "We are very pleased to announce the 1.5.0 release of Panel! While technically a minor release, this new release hugely expands the scope of what is possible in Panel. A high-level overview of the most important features:\n",
```

File: posts/panel_release_1.5/index.ipynb [hunk 3]
```diff
@@ -0,0 +1,2259 @@
...
+    "\n",
+    "ConfettiButton()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eb1b764b-db6c-409e-9336-ec9b42c834e9",
+   "metadata": {},
+   "source": [
+    "Breaking this down we can see how easy it is to create a DOM element, attach event listerns and finally, react to and update parameter values.\n",
```
Comment (jbednar) [inline posts/panel_release_1.5/index.ipynb:1349]:
```suggestion
    "Breaking this down we can see how easy it is to create a DOM element, attach event listeners and finally, react to and update parameter values.\n",
```

File: posts/panel_release_1.5/index.ipynb [hunk 4]
```diff
@@ -0,0 +1,2259 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "589b376d-e587-4f7d-8966-db24c256a29d",
+   "metadata": {},
+   "source": [
+    "### AnyWidget Compatibility\n",
+    "\n",
+    "At this point we also want to give a shoutout to the [`anywidget`](https://anywidget.dev/) project and particularly its author Trevor Mantz for some valuable discussion. It inspired many of the ideas behind these component classes and influenced the API. Additionally we also provide an [`AnyWidgetComponent`](https://panel.holoviz.org/reference/custom_components/AnyWidgetComponent.md) class which mirrors the JS API of `AnyWidget` making it possible to reuse AnyWidget components natively in Panel.\n",
```
Comment (jbednar) [inline posts/panel_release_1.5/index.ipynb:1496]:
```suggestion
    "At this point we also want to give a shoutout to the [`anywidget`](https://anywidget.dev/) project and particularly its author Trevor Manz for some valuable discussion. It inspired many of the ideas behind these component classes and influenced the API. Additionally we also provide an [`AnyWidgetComponent`](https://panel.holoviz.org/reference/custom_components/AnyWidgetComponent.md) class which mirrors the JS API of `AnyWidget` making it possible to reuse AnyWidget components natively in Panel.\n",
```

File: posts/panel_release_1.5/index.ipynb [hunk 5]
```diff
@@ -0,0 +1,2259 @@
...
+    "\n",
+    "To demonstrate this we will fetch the JS implementation of the [CarbonPlan widget](https://github.com/manzt/carbonplan-maps) directly from GitHub (don't do this) and implement only the Python wrapper class:"
```
Comment (jbednar) [inline posts/panel_release_1.5/index.ipynb:1498]: Can you clarify? E.g. "don't do this for real code", or...?

File: posts/panel_release_1.5/index.ipynb [hunk 6]
```diff
@@ -0,0 +1,2259 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ca5f3d39-750a-4a91-889e-0221cceb0580",
+   "metadata": {},
+   "source": [
+    "## Native FastAPI Integration\n",
+    "\n",
+    "FastAPI as a library is incredibly popular and we have received multipe requests to make it easier to integrate Panel with FastAPI. As of today that is a reality, by installing the additional package `bokeh-fastapi` you can now run Panel apps natively on a FastAPI server, e.g. using `uvicorn`.\n",
```
Comment (jbednar) [inline posts/panel_release_1.5/index.ipynb:1686]:
```suggestion
    "FastAPI as a library is incredibly popular and we have received multiple requests to make it easier to integrate Panel with FastAPI. As of today that is a reality! By installing the additional package `bokeh-fastapi`, you can now run Panel apps natively on a FastAPI server, e.g. using `uvicorn`.\n",
```

File: posts/panel_release_1.5/index.ipynb [hunk 7]
```diff
@@ -0,0 +1,2259 @@
...
+    "    return slider.rx() * '⭐'\n",
+    "```\n",
+    "\n",
+    "Now we can run the application with:\n",
+    "\n",
+    "```bash\n",
+    "fastapi dev main.py # or uvicorn main:app\n",
+    "```\n",
+    "\n",
+    "After visiting `http://localhost:8000/docs` you will see the following output:\n",
```
Comment (jbednar) [inline posts/panel_release_1.5/index.ipynb:1718]:
```suggestion
    "After visiting `http://localhost:8000/docs` you should see the following output:\n",
```

**Review by jbednar — APPROVED** [review]
Amazing new release!

**Comment by philippjfr** [comment]
@jbednar Thanks, applied your edits manually since merging notebook edits is a nightmare.

File: posts/panel_release_1.5/images/collage_1.5.png
Comment (MarcSkovMadsen) [inline posts/panel_release_1.5/images/collage_1.5.png:1]: Adding the AnyWidget logo to the picture below would make it even stronger.

File: posts/panel_release_1.5/index.ipynb [hunk 8]
```diff
@@ -0,0 +1,2331 @@
...
+    "\n",
+    "## New release!\n",
+    "\n",
+    "We are very pleased to announce the 1.5.0 release of Panel! While technically a minor release, this minor release hugely expands the scope of what is possible in Panel. A high-level overview of the most important features:\n",
+    "\n",
+    "- Make it trivially easy to build new JS and React based components with hot reload and built-in compilation and bundling\n",
+    "- Add native support for running Panel apps on a FastAPI server\n",
+    "- Add a number of new components including the `Placeholder` pane, `FileDropper` and `TimePicker` widgets, and `ChatStep` component.\n",
+    "- Greatly improved `ChatFeed` UX and support for streaming text chunks to the frontend.\n",
```
Comment (ahuang11) [inline posts/panel_release_1.5/index.ipynb:1236]:
```suggestion
    "- Greatly improved `ChatFeed` UX and support for streaming text chunks to the frontend.\n",
    "- More flexibility with what is displayed with the `Player` and `DiscretePlayer` widgets"
```

Comment (philippjfr) [reply posts/panel_release_1.5/index.ipynb]: I'm happy to mention it but I don't think it quite rises to a topline feature.

Comment (ahuang11) [reply posts/panel_release_1.5/index.ipynb]: Yeah a mention would be nice; previously, I felt like Player was too big to fit reasonably in places, or use in the sidebar.

Comment (philippjfr) [reply posts/panel_release_1.5/index.ipynb]: https://holoviz-dev.github.io/blog-dev/posts/panel_release_1.5/#player-and-discreteplayer

**Pull request merged**